readme.md

```diff
@@ -139,7 +143,13 @@ Define a Javascript constructor called 'Instructor'. Every instance of Instructo
 Instantiate an instructor named 'Andy' and call its `receivePresent` method with "Resin Laying Deer Figurine, Gold" as the argument.
 
 ```js
-// Your code starts here...
+var Instructor =
+[
+  name:
+  function: receivePresent(gift)
+    console.log( name + "promptly drops the"  +  gift + "on the floor." )
+
+]
 
 ```
 
```

Comment:
A constructor is a function. Not an object or array. As such, the basic format of a constructor looks something like this...

```js
var Instructor = function( name ){
  this.name = name;
}
```

While we could also include the `receivePresent` method in there, it's best practice to define it as a prototypal method so that we use up less memory every time we instantiate a new `Instructor`.

```js
Instructor.prototype.receivePresent = function( gift ){
  console.log( this.name + "promptly drops the " + gift + " on the floor." );
}
```